applications/nrf5340_audio/src/modules/audio_sync_timer.h

```diff
@@ -0,0 +1,29 @@
+/*
+ *  Copyright (c) 2021, PACKETCRAFT, INC.
```

Comment:
Hmm, the use of Packetcraft license here (and also in the .c file) is not optimal now, since these files contains both timer sync mechanisms.

Reply:
We've discussed offline that we may want to split this module into two different implementations. I guess we can update this later?

Reply:
We discussed this and will handle it in OCT-2783